src/components/Plate.vue

```diff
@@ -44,15 +44,15 @@
 // The plate structure is defined by the grid
 // Plate is made up of rows which are part of the grid which contains the wells.
 // When a plate is created it is added to the store as a sequencescape plate.
-// Once the plate is received from local storage a set of triplicates is created. The plate id is cascaded down so that the well can add a triplicate
+// Once the plate is received from local storage a set of replicate is created. The plate id is cascaded down so that the well can add a replicate
```

Comment:
```suggestion
// Once the plate is received from local storage a set of replicates is created. The plate id is cascaded down so that the well can add a replicate
```